MoneyMarketAccount.rb

```diff
@@ -0,0 +1,64 @@
+module Bank
+  class MoneyMarketAccount < Account
+    attr_reader :num_transactions, :transaction_allowed
+    def initialize(id, initial_balance, open_date, owner = nil)
+      @id = id
+      @balance = initial_balance
+      @min_balance = 1000000
+      @open_date = open_date
+      @owner = owner
+      @num_transactions = 0
+      @max_transactions = 6
+      @bal_above_min = true
+      check_balance
+    end
+
+    def withdraw(amount)
+      below_min_balance_fee = 10000
+      if @num_transactions >= @max_transactions
+        puts "You cannot make more than #{@max_transactions} transactions in one month. Transaction terminated."
+      else #number of transactions is less than the allotted maximum
+        if @bal_above_min
+          if (@balance - amount < @min_balance) #withdrawing the amount will put the account below the minimum balance
+            amount += below_min_balance_fee
+            puts "Your account is locked until you reach the minimum balance of $#{@min_balance}."
+            @bal_above_min = false
+          end
+            @balance -= amount
```

Comment:
Watch your indentation here - this code is outside of the `if` statement blocks so it does not need to have additional indentation